recipes/label_parsing.ipynb

```diff
@@ -0,0 +1,3063 @@
+{
```

Comment:
<p>When do you use entity_annotations? There's no example showing how to use this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>